src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts

```diff
@@ -668,13 +670,24 @@ class driveUsageSection {
         });
 
         items.forEach(incomingItem => {
-            const matched = this.table().find(x => x.mountPoint() === incomingItem.MountPoint);
+            const incomingItemMountPoint = incomingItem.MountPoint;
+            const matched = this.table().find(x => x.mountPoint() === incomingItemMountPoint);
+            
             if (matched) {
                 matched.update(incomingItem);
             } else {
-                const usage = new driveUsage(incomingItem, this.storageChart.getColorClassProvider(), this.includeTemporaryBuffers);
+                const usage = new driveUsage(incomingItem, this.includeTemporaryBuffers);
                 this.table.push(usage);
             }
+            
+            if (!(incomingItemMountPoint in this.numberOfDatabasesPerMountPoint)) {
```

Comment:
here I'd check like this:
if (this.numberOfDatabasesPerMountPoint[incomingItemMountPoint]) {

}

this will check if key exists and have positive value

Reply:
ok - will provide a fix shortly